packages/run-protocol/test/supports.js

```diff
@@ -84,7 +85,7 @@ export const setupBootstrap = (t, optTimer = undefined) => {
   brand.produce.RUN.resolve(runBrand);
   issuer.produce.RUN.resolve(runIssuer);
 
-  return { produce, consume, ...spaces };
+  return { produce, consume, modules: { utils: { ...utils } }, ...spaces };
```

Comment:
I wonder about adding ERTP and contractSupport to our list of available modules.